store/store_action.go

```diff
@@ -56,8 +57,8 @@ type CurrentSnap struct {
 	Block            []snap.Revision
 	Epoch            snap.Epoch
 	CohortKey        string
-	// ValidationSets is an optional array of validation sets primary keys.
-	ValidationSets [][]string
+	// ValidationSetKeys is an optional array of validation sets primary keys.
+	ValidationSetKeys []snapasserts.ValidationSetKey
```

Comment:
I think the idea here is that the field names try to match the JSON ones, so I think we should keep the old name for this. The type anyway helps knowing what is expected

Reply:
Renamed them back to `ValidationSets`